Doc/library/logging.rst

```diff
@@ -813,8 +813,9 @@ wire).
    :type lineno: int
 
    :param msg: The event description message,
-      which can be a %-format string with placeholders for variable data.
-   :type msg: str
+      which can be a %-format string with placeholders for variable data,
+      or an arbitrary object (see :ref:`arbitrary-object-messages`).
+   :type msg: str | Any
```

Comment:
This is a good type hint for typeshed, where the primary consumer of the type hints is a type checker. But it might seem strange for a reader of the CPython documentation, who might not be a typing expert. We could probably just use the slightly-less-precise-but-simpler type `Any` instead.

Also: we need to use `typing.Any` if we want a snazzy link to the docs for `typing.Any`, the same as we have in the docs for the `args` parameter: https://docs.python.org/3/library/logging.html#logging.LogRecord. You can see in the Netlify docs preview that we don't currently have a link for `Any` in this parameter's type hint in the same way: https://deploy-preview-102875--python-cpython-preview.netlify.app/library/logging.html?highlight=logging#logging.LogRecord 

```suggestion
   :type msg: typing.Any
```

Reply:
I did not know about this, thanks :+1: